client/src/pages/EducatorDashboard/EventCard.tsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import { makeStyles } from "@material-ui/core/styles";
+import Card from "@material-ui/core/Card";
+import CardContent from "@material-ui/core/CardContent";
+import Typography from "@material-ui/core/Typography";
+import Grid from "@material-ui/core/Grid";
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        flexGrow: 1,
+    },
+    card: {
+        margin: `${theme.spacing(7)}px auto`,
+        padding: theme.spacing(3),
+        borderRadius: 5
+    },
+    tag: {
+        padding: '0px 3px',
+        margin: '2px',
+        textAlign: 'center',
+        borderRadius: 10,
+        background: '#F7FAFC'
+    },
+    textHighlight: {
+        paddingLeft: '1em',
+        color: '#0A79BF',
+        fontWeight: 700
+    }
+
+}));
+
+function getDate(props: any) {
+
+    var eventStartDate = new Date(props.event.startDate).toLocaleString('default', { month: 'long', day: 'numeric', year: 'numeric' }).replace(',', '')
+    var eventEndDate = new Date(props.event.endDate).toLocaleString('default', { month: 'long', day: 'numeric', year: 'numeric' }).replace(',', '')
+
+    if (eventStartDate === eventEndDate) {
+        var date = eventStartDate
```

Comment:
instantiate date outside of control flow statement so we don't have to redeclare

+ remove console log line 47